### PR TITLE
Updated guide to use the new tailscale funnel commands

### PR DESCRIPTION
**Pull request by arjanflac**
Updated guide to use the new tailscale funnel commands & added info to use on host system when using a virtutal macOS

### CODE REVIEW

File: server/advanced/byo-proxy-service-guides/tailscale-vpn-setup.md [hunk 1]
```diff
@@ -1,20 +1,51 @@
+# Tailscale Funnel 
+**Requirements**
+
+- Enable [HTTPS certificates](https://login.tailscale.com/admin/dns) in the [admin console](https://login.tailscale.com/admin/dns).
+- [Add Funnel to Policy](https://login.tailscale.com/admin/acls) in the admin console.
+
 ---
-description: >-
```
Comment (jjoelj) [inline server/advanced/byo-proxy-service-guides/tailscale-vpn-setup.md:2]: I think you should keep some sort of description directly under the first header to keep the format the same as the other guides. It gives some context about what prerequisites there are to use this guide (e.g. having a Tailscale account). The distinction of this being an option that doesn't involve port-forwarding is also important.

File: server/advanced/byo-proxy-service-guides/tailscale-vpn-setup.md [hunk 2]
```diff
@@ -1,20 +1,51 @@
...
+tailscale serve status
+```
+
+6. Finally, add the domain from step 5 to the BlueBubbles Proxy Service drop-down menu:
+
+```bash
+<https://machine-name.example.ts.net:443/>
+```
+--- 
+## Run on Host System |  macOS virutal machine
```
Comment (jjoelj) [inline server/advanced/byo-proxy-service-guides/tailscale-vpn-setup.md:42]: Could you clarify what this header means and/or reword it.

**Review by jjoelj — CHANGES_REQUESTED** [review]
Great! Minor changes requested.

File: server/advanced/byo-proxy-service-guides/tailscale-vpn-setup.md [hunk 3]
```diff
@@ -1,20 +1,52 @@
...
+```bash
+echo 'alias tailscale="/Applications/Tailscale.app/Contents/MacOS/Tailscale"' | sudo tee -a ~/.zshrc
+```
+Alternatively, you can use `/Applications/Tailscale.app/Contents/MacOS/Tailscale <command>` 
+
+3. Proxy requests to BlueBubbles's local web server on port 1234.
+
+```bash
+tailscale serve https / http://127.0.0.1:1234
+```
```
Comment (tneotia) [inline server/advanced/byo-proxy-service-guides/tailscale-vpn-setup.md:25]: Can you clarify that this port # could be different depending on what the user has set up (but indicate that its usually 1234)

File: server/advanced/byo-proxy-service-guides/tailscale-vpn-setup.md [hunk 4]
```diff
@@ -1,20 +1,52 @@
...
+5. Check the funnel status with:
+
+```bash
+tailscale serve status 
+```
+
+6. Finally, add the domain from step 5 to the BlueBubbles Proxy Service drop-down menu:
+
+```bash
+<https://machine-name.example.ts.net:443/>
```
Comment (tneotia) [inline server/advanced/byo-proxy-service-guides/tailscale-vpn-setup.md:42]: I don't think the <> are needed here

**Review by tneotia — CHANGES_REQUESTED** [review]
LGTM otherwise

**Pull request merged**